localstack/services/awslambda/lambda_api.py

```diff
@@ -1375,6 +1494,122 @@ def add_permission(function):
+
+    data = json.loads(to_str(request.data))
+    cors = data.get("Cors", {})
+    url_config = {
+        "AuthType": data.get("AuthType"),
+        "Cors": {
+            "AllowCredentials": cors.get("AllowCredentials", ["*"]),
+            "AllowHeaders": cors.get("AllowHeaders", ["*"]),
+            "AllowMethods": cors.get("AllowMethods", ["*"]),
+            "AllowOrigins": cors.get("AllowOrigins", ["*"]),
+            "ExposeHeaders": cors.get("ExposeHeaders", []),
+            "MaxAge": cors.get("MaxAge", 0),
+        },
+        "FunctionArn": arn,
+        "FunctionUrl": url,
+        "CreationTime": datetime.now().isoformat(),
+        "CustomId": custom_id,
+    }
+
+    lambda_backend.url_configs.update({arn: url_config})
+    response = url_config.copy()
+    response.pop("CustomId")
+    return response
+
+
+@app.route("%s/functions/<function>/url" % API_PATH_ROOT_2, methods=["GET"])
+def get_url_config(function):
+    arn = func_arn(function)
+    qualifier = request.args.get("Qualifier")
+    q_arn = func_qualifier(function, qualifier)
+    arn = q_arn or arn
+    lambda_backend = LambdaRegion.get()
+    url_config = lambda_backend.url_configs.get(arn)
+
+    if url_config is None:
+        return not_found_error()
+
+    response = url_config.copy()
+    response.pop("CustomId")
+    return response
+
+
+@app.route("%s/functions/<function>/url" % API_PATH_ROOT_2, methods=["PUT"])
+def update_url_config(function):
+    arn = func_arn(function)
+    qualifier = request.args.get("Qualifier")
+    q_arn = func_qualifier(function, qualifier)
+    arn = q_arn or arn
+
+    lambda_backend = LambdaRegion.get()
+    prev_url_config = lambda_backend.url_configs.get(arn)
+
+    if prev_url_config is None:
+        return not_found_error()
+
+    data = json.loads(to_str(request.data))
+    cors = data.get("Cors", {})
+    new_url_config = {
+        "AuthType": data.get("AuthType"),
+        "Cors": {
+            "AllowCredentials": cors.get("AllowCredentials", ["*"]),
+            "AllowHeaders": cors.get("AllowHeaders", ["*"]),
+            "AllowMethods": cors.get("AllowMethods", ["*"]),
+            "AllowOrigins": cors.get("AllowOrigins", ["*"]),
+            "ExposeHeaders": cors.get("ExposeHeaders", []),
+            "MaxAge": cors.get("MaxAge", 0),
+        },
+        "LastModifiedTime": datetime.now().isoformat(),
+    }
+    prev_url_config.update(new_url_config)
+
+    response = prev_url_config.copy()
+    response.pop("CustomId")
+    return response
+
+
+@app.route("%s/functions/<function>/url" % API_PATH_ROOT_2, methods=["DELETE"])
+def delete_url_config(function):
+    arn = func_arn(function)
+    qualifier = request.args.get("Qualifier")
+    q_arn = func_qualifier(function, qualifier)
+    arn = q_arn or arn
+
+    lambda_backend = LambdaRegion.get()
+    if arn in lambda_backend.url_configs.keys():
+        lambda_backend.url_configs.pop(arn)
```

Comment:
```suggestion
    lambda_backend.url_configs.pop(arn, None)
```
could be replaced like this, no if check necessary.